.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Feature request
+about:  Would you like to suggest a feature? Please use this template. 
+title: 'Feature request: '
+assignees: ''
```

Comment:
I'm not familiar with Issue templates.  Will this `assignees` entry prompt the user for an assignee?

Reply:
Unfortunately no, afaik. It leaves an issue unassigned and we can specify assignees per template if we would like

Reply:
If we would like to assign people automatically we can use the CODEOWNERS file of GitHub. 

https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners